src/otx/algo/detection/__init__.py

```diff
@@ -1,6 +1,6 @@
-"""Custom model implementations for detection task."""
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
+"""Custom model implementations for detection task."""
```

Comment:
Based on my comment in main PR, I think this should stay as is, but it could be done in another PR since there are quite a bit of files to be changed :) --We also need to agree on the option first. 

Common practice in the python community though is the following:

```python
"""Custom model implementations for detection task."""

# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

from . import backbones, heads

...
```

Reply:
Yeah, we all agree that. Let's keep it AS-IS and then change all ones at one shot